### ISSUE TITLE
selects at most one floating block?

### ISSUE THREAD

**Issue by kzidane** [cs50/harvard.cs50.droplet#67]
![selection](https://user-images.githubusercontent.com/7230211/28798978-8805002c-7614-11e7-9d1e-461092976c24.gif)

Is this the expected behavior?

**Comment by dabbler0** [comment]
Yes, this is the expected behavior. There are a lot of droplet mechanics that don't make sense if you select two blocks that aren't connected to each other.

**Comment by dmalan** [comment]
Do we need the lasso behavior at all? Does it not suffice just to expect students to click on individual blocks, given that constraint?

**Comment by dabbler0** [comment]
Lasso behavior isn't strictly necessary but it can be useful if students, for instance, have something like:

```
if (x) {
  y;
  z;
  w;
}
```

And want to remove the "if x" condition; without lasso select they'd have to move each x, y, z, out individually, whereas with lasso select they can select the entire sequence and move it out of the `if` in one step.

But if you think it's confusing we can disable this.
